cmd/subctl/azure.go

```diff
@@ -16,28 +16,36 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// nolint:dupl // Similar code in aws.go, azure.go, rhos.go, but not duplicate
```

Comment:
Just a question, why not do the flag init once for all `cloud` commands? They all need the same set of flags.

Reply:
Fair question, I’ve updated the PR to share the setup. Thanks!